src/PhpOption/Option.php

```diff
@@ -399,8 +413,8 @@ public static function lift($callback, $noneValue = null)
      *
      * @template S
      *
-     * @param S        $initialValue
-     * @param callable $callable     function(initialValue, callable): result
+     * @param S                        $initialValue
+     * @param callable(mixed, mixed):S $callable     function(callable, initialValue): result
```

Comment:
Is the first param of the callable mixed?

Reply:
I don’t know, but better to annotate that a param is required than to have no param at all

Reply:
It probably should be `callable(S, S):S`, I think, as that's how `Some` and `None` behave.